src/client/datascience/notebook/notebookEditor.ts

```diff
@@ -95,6 +97,15 @@ export class NotebookEditor implements INotebookEditor {
         );
         disposables.push(model.onDidDispose(this._closed.fire.bind(this._closed, this)));
     }
+    @captureTelemetry(Telemetry.SyncAllCells)
+    public async syncAllCells(): Promise<void> {
+        // Need to read content out of the UI and save in our model
+        const modifiedCells = (this.model as VSCodeNotebookModel).getCells().map((c, i) => {
+            const dc = this.document.cells[i];
```

Comment:
Curious, why do we need this for `native notebooks`?
Was that failing as well?

Reply:
Yes, but only for save, not for execute. See this comment here:
https://github.com/microsoft/vscode-jupyter/issues/1701#issuecomment-738929286